apps/updownload/src/components/Import.vue

```diff
@@ -179,8 +183,14 @@ export default {
           fetch(url, options)
             .then((response) => {
               if (response.ok) {
-                response.text().then((successText) => {
-                  this.success = successText;
+                response.json().then((response) => {
```

Comment:
More of a preference really, but in the `response.ok` check can also return `response.json()` and pass it to another `then`.

```
fetch(...)
.then((response) => {
    if (response.ok) {
       return response.json()
    }
  ...
})
.then((result) => {
   // process the response here
})
.catch((err) => {
   // handle error here
})
```